src/api/goods.ts

```diff
@@ -10,10 +10,11 @@ export function getAll(): Promise<Good[]> {
 
 export const get5First = () => {
   return getAll()
-    .then(goods => goods); // sort and get the first 5
+    .then((goods) => [...goods].sort((a, b) => a.name.localeCompare(b.name))
```

Comment:
You don't need to create the copy here. You will not mutate the promise `result`